specification/eventgrid/resource-manager/readme.md

```diff
@@ -0,0 +1,40 @@
+# EventGrid
+
+> see https://aka.ms/autorest
+
+This is the AutoRest configuration file for Azure EventGrid.
+
+
+
+---
+## Getting Started
+To build the SDK for Azure EventGrid, simply [Install AutoRest](https://aka.ms/autorest/install) and in this folder, run:
+
+> `autorest`
+
+To see additional help and options, run:
+
+> `autorest --help`
+---
+
+## Configuration
+
+
+
+### Basic Information
+These are the global settings for the Azure EventGrid API.
+
+``` yaml
+openapi-type: arm
+tag: package-2017-06-preview
```

Comment:
nit: we should probably keep tags consistent with the name of the directory in which the json is placed.